src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -52,18 +52,17 @@
     <Compile Include="System\Text\Json\Writer\Utf8JsonWriter.WriteValues.String.cs" />
     <Compile Include="System\Text\Json\Writer\Utf8JsonWriter.WriteValues.UnsignedNumber.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\..\System.Collections\src\System.Collections.csproj" />
-    <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj" />
-    <ProjectReference Include="..\..\System.Diagnostics.Tools\src\System.Diagnostics.Tools.csproj" />
+  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
     <ProjectReference Include="..\..\System.Memory\src\System.Memory.csproj" />
-    <ProjectReference Include="..\..\System.Numerics.Vectors\src\System.Numerics.Vectors.csproj" />
-    <ProjectReference Include="..\..\System.Resources.ResourceManager\src\System.Resources.ResourceManager.csproj" />
-    <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
     <ProjectReference Include="..\..\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj" />
-    <ProjectReference Include="..\..\System.Text.Encoding.Extensions\src\System.Text.Encoding.Extensions.csproj" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
```

Comment:
If you keep this reference to CoreLib, this package will be partially inboxed and thus immediately dead.

Reply:
Unresolving.

Based on feedback from @ericstj (see https://github.com/dotnet/corefx/pull/34552#issuecomment-454924961), and discussions with @joperezr, I will be adding netstandard as a build configuration (for testing and validation) and conditionally include S.Private.Corelib. However, it won't be included in the package proj and we won't ever ship a netstandard binary produced from that configuration.

@jkotas Why would that result in the S.T.Json package to be partially inboxed?

The plan is to define a custom constant (`BuildingForInboxLibrary`) which is true when `TargetGroup != netstandard` and bring these conditional includes back (essentially revert this commit - https://github.com/dotnet/corefx/pull/34611/commits/7a2f22e127c4ef0476f20bddd64796045f79ddb5)

Reply:
> Why would that result in the S.T.Json package to be partially inboxed?

This comment was assuming that you are building a proper binary package. You are not doing it (yet...).